### DISCUSSION TITLE
how to round .FormatDuration

### DISCUSSION THREAD

**Discussion by scialo** [garethgeorge/backrest#292] [General]
In my notification service gotify i set my template for duration: 
`- Total duration: {{ .FormatDuration .Duration }}`

when i display the notify on gotify i read:
`- Total duration: 53.5859536s`

how to round 54s like in the report of backrest?

thanks

**Comment by garethgeorge** [comment]
Fixed this in a patch, I don't plan on offering a way to fully customize this (though if you care deeply I'd accept a PR) but I've switched the default to round to milliseconds.

**Comment by scialo** [comment]
thanks
